src/mongocxx/collection.hpp

```diff
@@ -350,6 +350,10 @@ class MONGOCXX_API collection {
     /// @return The original or replaced document.
     /// @throws exception::write if the operation fails.
     ///
+    /// @note
```

Comment:
I see. I don't think that C driver can reasonably decline both this request and the "copy a collection/database" request. Might want to point that out to them.